### PR TITLE
P20-714: Prevent uploading the `animations` localized manifest during sync-out when testing

### PR DESCRIPTION
**Pull request by artem-vavilov**
The AWS S3 instance, to which we upload the `animations` localized manifest, is used in production.
So we must prevent the uploading of any testing data to it.

- jira ticket: [P20-714](https://codedotorg.atlassian.net/browse/P20-714)

### CODE REVIEW

File: bin/i18n/sync-all.rb [hunk 1]
```diff
@@ -98,11 +98,12 @@ def sync_out
   end
```
Comment (mgc1194) [inline bin/i18n/sync-all.rb:98]: Should we switch to this?
```suggestion
  def sync_out
    I18n::SyncOut.perform(options.slice(:testing))
  end
```

File: bin/i18n/sync-all.rb [hunk 2]
```diff
@@ -130,12 +131,7 @@ def parse_options(args)
       opts.on("-y", "--yes", "Run without confirmation") do
         options[:yes] = true
       end
```
Comment (mgc1194) [inline bin/i18n/sync-all.rb:133]: I wonder if we should move the rest of the options to the one parser. On one hand, having one parsing method could be cleaner, on the other hand, most options are only used in the `sync-all` script, and could get confusing to have options available that are not used in other modules.

Comment (artem-vavilov) [reply bin/i18n/sync-all.rb:133]: We shouldn't add options that we are not going to use in the scripts

File: bin/i18n/i18n_script_utils.rb [hunk 3]
```diff
@@ -28,6 +28,24 @@ def self.crowdin_creds
...
+      parser.on('-t', '--testing', 'Run in testing mode') do
+        options[:testing] = true
+      end
+
+      yield(parser, options) if block_given?
+    end.parse!(argv)
+
+    options
+  end
+
```
Comment (mgc1194) [inline bin/i18n/i18n_script_utils.rb:48]: When I read the description of the PR moving the `option parser` to the `i18n_script_utils` was my first thought. I'm really glad we were thinking the same way!

**Review by mgc1194 — APPROVED** [review]
LGTM!!

**Pull request merged**